safe_authenticator/src/client.rs

```diff
@@ -154,52 +154,54 @@ where {
         let pub_key = PublicKey::from(maid_keys.bls_pk);
         let full_id = Some(maid_keys.clone().into());
 
+        let acc = Account::new(maid_keys.clone())?;
+
+        let acc_ciphertext = acc.encrypt(&user_cred.password, &user_cred.pin)?;
+
+        let mut seeder: Vec<u8> = Vec::with_capacity(acc_locator.len() + acc_password.len());
+        seeder.extend_from_slice(acc_locator);
+        seeder.extend_from_slice(acc_password);
+
+        let seed = sha3_256(&seeder);
+        let mut rng = StdRng::from_seed(seed);
+        let client_full_id = ClientFullId::new_bls(&mut rng);
+
+        let sig = client_full_id.sign(&acc_ciphertext);
+        let client_pk = client_full_id.public_id().public_key();
+        let new_account_data = LoginPacket::new(acc_loc, *client_pk, acc_ciphertext, sig)?;
+        let balance_client_id = ClientFullId::with_bls_key(balance_sk);
+
+        {
+            let (mut routing, routing_rx) = setup_routing(
+                full_id.clone(),
+                PublicId::Client(balance_client_id.public_id().clone()),
+                None,
+            )?;
+
+            let rpc_response = routing.req_as_client(
+                &routing_rx,
+                Request::CreateLoginPacketFor {
+                    new_owner: pub_key,
+                    amount: unwrap!(Coins::from_str("1")),
```

Comment:
Although it's temp code, I think we shouldn't use unwrap here:
```suggestion
                    amount: Coins::from_str("1")?
```
Even better, take the `amount` as an argument for this function.